cmd/res/configuration.yaml

```diff
@@ -0,0 +1,47 @@
+# REFERENCE: https://docs.edgexfoundry.org/3.0/microservices/device/Ch-DeviceServices/
+Writable:
+  LogLevel: INFO
+  Reading:
+    ReadingUnits: true
+  Telemetry:
+    Metrics:
+      EventsSent: false
+      ReadingsSent: false
+
+Service:
+  Host: localhost
+  Port: 59995
+  StartupMsg: device uart started
+  RequestTimeout: "5s"
+
+MessageBus:
+  Protocol: redis
+  Host: localhost # uncomment when running from command-line in hybrid mode
+  Port: 6379
+  Type: redis
+  BaseTopicPrefix: "edgex" # prepended to all topics as "edgex/<additional topic levels>  
+  Optional:
+    ClientId: device-uart
+
+Clients:
+  core-metadata:
+    Protocol: "http"
+    Host: "localhost"
+    Port: 59881
+
+Device:
+  DataTransform: true
+  MaxCmdOps: 128
+  MaxCmdValueLen: 256
```

Comment:
Remove, these are in the new common config here: https://github.com/edgexfoundry/edgex-go/blob/main/cmd/core-common-config-bootstrapper/res/configuration.yaml#L117C8-L130C22